<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>254</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Poor Richard</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LogIn Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="account_username"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="account_sign_in">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>SignIn</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="account_password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="clear_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="Player1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player 1</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QRadioButton" name="Player2">
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player 2</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
